GitOut/Features/Wpf/Converters/ColorToBrushConverter.cs

```
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace GitOut.Features.Wpf.Converters
+{
+    public class ColorToBrushConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+            => value is Color color ? new SolidColorBrush(color) : DependencyProperty.UnsetValue;
```

This should probably throw if value is not a color, rather than silently succeeding

---

anticipated errors [should return `DependencyProperty.UnsetValue`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.ivalueconverter.convert?redirectedfrom=MSDN&view=netframework-4.8#remarks)